### DISCUSSION TITLE
Dependency incompatibility issue creates LS to be stopped

### DISCUSSION THREAD

**Discussion by dulajdilshan** [ballerina-platform/ballerina-lang#34228] [General]
`Dependencies.toml` file is a file generated automatically, but when a user changes the version, an exception is thrown. 

Related Issues: 

1. https://github.com/ballerina-platform/ballerina-lang/issues/34221
2. https://github.com/ballerina-platform/ballerina-lang/issues/33765

**Comment by dulajdilshan** [comment]
An exception is thrown when there are two different versions. ([Please refer to this comment)](https://github.com/ballerina-platform/ballerina-lang/issues/34221#issuecomment-989505112)

**Reply by nadeeshaan** [reply]
@azinneera please share your thoughts as well
